scout/server/blueprints/variant/templates/variant/variant_details.html

```diff
@@ -165,24 +165,36 @@
                <td>{{ obs.homozygote|default('N/A') }}</td>
                <td>{{ obs.total|default('N/A') }}</td>
              </tr>
-           {% if obs.cases %}
+           {% if obs.cases or obs.observations == 1 %}
              <tr>
                <td colspan="4">
-                     {% for data in obs.cases %}
-                       {% if data.variant and data.variant.category == "snv"%}
-                         <a class="badge rounded-pill bg-light text-dark" target="_blank" href="{{ url_for('variant.variant', institute_id=data.case.owner, case_name=data.case.display_name, variant_id=data.variant._id) }}">{{ data.case.display_name }}</a>
-                       {% elif data.variant and data.variant.category == "sv"%}
-                         <a class="badge rounded-pill bg-light text-dark" target="_blank" href="{{ url_for('variant.sv_variant', institute_id=data.case.owner, case_name=data.case.display_name, variant_id=data.variant._id) }}">{{ data.case.display_name }}</a>
-                       {% else %}
-                         <span class="ml-3 badge rounded-pill bg-light text-dark">{{ data.case.display_name }}</span>
-                       {% endif %}
-                     {% endfor %}
-                     {% if obs.cases|length > 10 %}
-                       + more
-                     {% endif %}
-                     {% if obs.observations == 1 %}
-                       Observed only in this case
-                     {% endif %}
+                 {% if obs.observations == 1 %}
+                 {% if obs.case_match==True %}
+                 Observed only in this case
+                 <span id="case_match_tooltip"  data-bs-toggle='tooltip' title="Subsequent analysis of the same individual or multiple occurrences within the same family will usually not result in an increased observation count, even if performed by different institutes.">
+                   <i class="mx-1 fas fa-info-circle"></i>
+                 </span>
+                 {% elif obs.case_match ==False %}
```

Comment:
```suggestion
                 {% elif obs.case_match == False %}
```
Being consistent with whitespaces around operators is nice, but hey, its not like we always do.